### DISCUSSION TITLE
我配置错误在什么地方呀？

### DISCUSSION THREAD

**Discussion by moshangchenkai** [MasterYip/ChatPaper2Xmind#2] [Q&A]
(base) C:\Users\mosha>cd D:\academic chatgpt series\ChatPaper2Xmind-main

(base) C:\Users\mosha>cd /d D:\academic chatgpt series\ChatPaper2Xmind-main

(base) D:\academic chatgpt series\ChatPaper2Xmind-main>python paper2xmind.py --path "C:\Users\mosha\Downloads\研学旅游视阈下革命纪念馆爱国主义教育实践探索——以刘少奇同志纪念馆为例.pdf"
Traceback (most recent call last):
  File "D:\academic chatgpt series\ChatPaper2Xmind-main\paper2xmind.py", line 8, in <module>
    from XmindCopilot import xmind, fileshrink
ImportError: cannot import name 'xmind' from 'XmindCopilot' (unknown location)

(base) D:\academic chatgpt series\ChatPaper2Xmind-main>python paper2xmind.py
Traceback (most recent call last):
  File "D:\academic chatgpt series\ChatPaper2Xmind-main\paper2xmind.py", line 8, in <module>
    from XmindCopilot import xmind, fileshrink
ImportError: cannot import name 'xmind' from 'XmindCopilot' (unknown location)

(base) D:\academic chatgpt series\ChatPaper2Xmind-main>

**Comment by MasterYip** [comment]
请确保此条命令执行正确：
git submodule update --init --recursive

另外，目前可能对中文论文、单栏论文的支持并不好。😂
